ui/admin/app/components/form/onboarding/index.hbs

```diff
@@ -0,0 +1,60 @@
+<Rose::Form
+  class='full-width'
+  @cancel={{@cancel}}
+  @disabled={{@model.target.isSaving}}
+  @onSubmit={{fn @submit this.targetAddress this.targetPort}}
+  as |form|
+>
+  <div class='create-resources-form'>
+    <Hds::Form::TextInput::Field
+      name='targetAddress'
+      @type='text'
+      @value={{this.targetAddress}}
+      @isRequired={{true}}
+      @isInvalid={{this.targetAddress}}
+      disabled={{form.disabled}}
+      {{on 'input' (set-from-event this 'targetAddress')}}
+      as |F|
+    >
+      <F.Label>{{t 'onboarding.add-target.titles.target-address'}}</F.Label>
+      <F.HelperText>{{t
+          'onboarding.add-target.descriptions.target-address'
+        }}</F.HelperText>
+    </Hds::Form::TextInput::Field>
+
+    <Hds::Form::TextInput::Field
+      name='targetPort'
+      @type='text'
+      @value={{this.targetPort}}
+      @isRequired={{true}}
+      {{on 'input' (set-from-event this 'targetPort')}}
+      as |F|
+    >
+      <F.Label>{{t 'onboarding.add-target.titles.target-port'}}</F.Label>
+      <F.HelperText>{{t
+          'onboarding.add-target.descriptions.target-port'
+        }}</F.HelperText>
+    </Hds::Form::TextInput::Field>
+  </div>
+
+  <form.fieldset>
```

Comment:
can we replace this with `<Hds::Form::Fieldset>`

Reply:
@DhariniJeeva Good comment. Initially I though to do so but didn't find any usage of `<Hds::Form::Fieldset>` with our codebase so I though to keep it simple. I changed the inputs to be HDS so I will try to get the `<Hds::Form::Fieldset>` that will be nice.
Thanks for the suggestion 😉

Reply:
@calcaide Originally we did not have an example of the `<HDS::Form::Fieldset>` but we do now! We added it during our multi-hop work. See the example [here](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/components/form/target/details/index.hbs#L180).

Reply:
We recently replaced HDS fieldset for [target forms](https://github.com/hashicorp/boundary-ui/blob/9b998ece1b04fb35091b1991aa8048828a850cfb/ui/admin/app/components/form/target/details/index.hbs#L180)

Reply:
Thanks for the references I was not aware from your work with `<HDS>>Form>>Fieldset>`. I also found interesting [its documentation](https://hds-website-hashicorp.vercel.app/components/form/primitives?tab=code) I was not obvious but found it 😉 
Changes applied.